open-metadata-implementation/governance-servers/data-engine-proxy-services/data-engine-proxy-services-server/src/main/java/org/odpi/openmetadata/governanceservers/dataengineproxy/auditlog/DataEngineProxyAuditCode.java

```diff
@@ -66,11 +61,17 @@
             "The user is not authorized for the Data Engine OMAS operation: {0}",
             "The system is unable to process the operation due to the user not being authorized to do so.",
             "Check your OMAS configuration and user authorizations."),
-    UNKNOWN_ERROR("DATA-ENGINE-PROXY-0012",
-            OMRSAuditLogRecordSeverity.EXCEPTION,
-            "An unknown error occurred",
-            "The system is unable to process the operation due to an unknown runtime error.",
-            "Check your OMAS configuration and server logs to troubleshoot."),
+    POLLING_TYPE_START("DATA-ENGINE-PROXY-0012",
```

Comment:
would this result in an audit log every 60s? I wonder if that makes it quite noisy - though maybe that's ok given we can control destination/filter